docs/evm-sidechain/xrpl-evm-sidechain-faq.md

```diff
@@ -0,0 +1,104 @@
+
+EVM compatibility on different blockchains:
+
+| Blockchain Ecosystem | EVM Compatibility Solution/Project |
+|----------------------|------------------------------------|
+| Ethereum             | Native                             |
+| Solana               | Neon                               |
+| Polkadot             | Moonbeam                           |
+| Cosmos               | Evmos                              |
+| Polygon              | zkEVM                              |
+| BNB Chain            | BNB Smart Chain                    |
+| Avalanche            | Avalance C-chain                   |
+| **XRPL**             | **EVM Sidechain**                  |
+
+An EVM sidechain will help the XRPL ecosystem attract EVM developers who can deploy their existing EVM applications on the EVM sidechain with minor modifications.
+
+
+### What are the main benefits of the EVM sidechain?
+
+The main benefits are:
+
+- Faster time to launch by deploying applications as smart contracts.
+- No amendment approval needed to launch applications.
+- Make minor changes to existing smart contracts on other blockchains and launch them on the EVM sidechain.
+- Grow the XRPL ecosystem by adding EVM ecosystem projects and developers.
+
+
+### What are the main risks with the EVM sidechain?
+
+With the XRPL, developers use existing primitives such as payments, DEX, escrow etc. which are secure and have a great track record. With EVM applications, the security is the responsibility of the smart contract developer. Smart contract bugs can get exploited and they have led to a loss of funds.
+
+
+### What is the main use case for the EVM sidechain?
+
+From an XRPL community perspective, the EVM sidechain enables EVM developers to launch their existing applications in the XRPL ecosystem. Developers can now write smart contracts and build applications, and not be constrained by the capabilities of XRPL smart transactors. XRPL developers can launch applications that previously required XRPL protocol change and an amendment proposal which is complex, resource intensive, and risky.
+
+
+### How many EVM sidechains will there be? Can I set up my own?
+
+There will be one public EVM sidechain supported by Ripple and PeerSyst for the XRPL community. This EVM sidechain will have multiple validators run by XRPL community members. The software needed to run an EVM sidechain will be open source and any community member can use it to set up their own EVM sidechains, for permissioned use cases for example.
```

Comment:
I'm curious about the future tense, here. We don't speculate about future features. Is there going to be a public EVM sidechain during the amendment process?